website/content/docs/concepts/iam.mdx

```diff
@@ -41,6 +41,10 @@ method's identity provider to provide up-to-date information.
 [permissions](/boundary/docs/concepts/security/permissions) which are granted to any principal (user or group) from any scope. Roles
 can be contained by any scope, and the permissions can be applied to the same scope or any child scope.
 
+<Tip>
```

Comment:
@Dan-Heath would it be appropriate to change this into some kind of note saying that we want to change this to support multiple roles in the future? This is a minor change we publicly comment on, not sure if it is acceptable in documentation

Reply:
I'd recommend against describing future plans like this in the docs. It could potentially be confusing to describe the way we want a feature to work in the future. Readers might not remember the future caveat later and mistakenly think it is supported now. Also, the text itself can become dated quickly if the update doesn't happen according to our anticipated schedule.

Reply:
Makes sense, won't put that in